labgrid/remote/client.py

```diff
@@ -894,6 +894,28 @@ def tmc_channel(self):
         for k, v in sorted(data.items()):
             print("{:<16s} {:<10s}".format(k, str(v)))
 
+    def write_image(self):
+        place = self.get_acquired_place()
+        target = self._get_target(place)
+        drv = None
+        from ..resource.remote import NetworkUSBMassStorage, NetworkUSBSDMuxDevice
+        from ..driver import NetworkUSBStorageDriver
+        try:
+            drv = target.get_driver(NetworkUSBStorageDriver)
+        except NoDriverFoundError:
+            for resource in target.resources:
+                if isinstance(resource, (NetworkUSBSDMuxDevice, NetworkUSBMassStorage)):
+                    try:
+                        drv = target.get_driver(NetworkUSBStorageDriver)
+                    except NoDriverFoundError:
+                        drv = NetworkUSBStorageDriver(target, name=None)
+                    drv.storage.timeout = self.args.wait
+                    break
+        if not drv:
+            raise UserError("target has no compatible resource available")
+        target.activate(drv)
+        drv.write_image(self.args.filename)
```

Comment:
```suggestion
        try:
            drv.write_image(self.args.filename)
        except subprocess.CalledProcessError as e:
            raise UserError("Could not write image to network usb storage: {}".format(e))
```

Reply:
Nice catch with the `FileNotFoundError` raised by the `ManagedFile`.